content/4_Gold/Ternary_Search.mdx

```diff
@@ -0,0 +1,146 @@
+---
+id: ternary-search
+title: Ternary Search
+author: Chongtian Ma
+prerequisites:
+  - /silver/binary-search
+redirects:
```

Comment:
can you remove these redirects?

Reply:
i copied from some module but ok 💀

Reply:
that's for when the module was originally in one division but was moved